draft-ietf-quic-http.md

```diff
@@ -450,15 +457,103 @@ field names MUST be converted to lowercase prior to their encoding.  A request
 or response containing uppercase header field names MUST be treated as
 malformed ({{malformed}}).
 
+Like HTTP/2, HTTP/3 does not use the Connection header field to indicate
+connection-specific header fields; in this protocol, connection- specific
+metadata is conveyed by other means.  An endpoint MUST NOT generate an HTTP/3
+message containing connection-specific header fields; any message containing
+connection-specific header fields MUST be treated as malformed ({{malformed}}).
+
+The only exception to this is the TE header field, which MAY be present in an
+HTTP/3 request; when it is, it MUST NOT contain any value other than "trailers".
+
+This means that an intermediary transforming an HTTP/1.x message to HTTP/3 will
+need to remove any header fields nominated by the Connection header field, along
+with the Connection header field itself.  Such intermediaries SHOULD also remove
+other connection-specific header fields, such as Keep-Alive, Proxy-Connection,
+Transfer-Encoding, and Upgrade, even if they are not nominated by the Connection
+header field.
+
+#### Pseudo-Header Fields
+
 As in HTTP/2, HTTP/3 uses special pseudo-header fields beginning with the ':'
 character (ASCII 0x3a) to convey the target URI, the method of the request, and
-the status code for the response.  These pseudo-header fields are defined in
-Section 8.1.2.3 and 8.1.2.4 of {{!HTTP2}}. Pseudo-header fields are not HTTP
-header fields.  Endpoints MUST NOT generate pseudo-header fields other than
-those defined in {{!HTTP2}}.  The restrictions on the use of pseudo-header
-fields in Section 8.1.2 of {{!HTTP2}} also apply to HTTP/3.  Messages which
-are considered malformed under these restrictions are handled as described in
-{{malformed}}.
+the status code for the response.
+
+Pseudo-header fields are not HTTP header fields.  Endpoints MUST NOT generate
+pseudo-header fields other than those defined in this document, except as
+negotiated via an extension; see {{extensions}}.
+
+Pseudo-header fields are only valid in the context in which they are defined.
+Pseudo-header fields defined for requests MUST NOT appear in responses;
+pseudo-header fields defined for responses MUST NOT appear in requests.
+Pseudo-header fields MUST NOT appear in trailers.  Endpoints MUST treat a
+request or response that contains undefined or invalid pseudo-header fields as
+malformed ({{malformed}}).
```

Comment:
This probably needs to be forcibly rewrapped -- obviously the editor's auto-wrapping was confused by the spacing.

Reply:
Nope, it actually does wrap this way by sheer coincidence.  (Or @martinthomson was playing a really long prank in RFC 7540.)